package.json

```
@@ -43,10 +45,12 @@
   "devDependencies": {
     "@typescript-eslint/eslint-plugin": "^5.42.0",
     "@typescript-eslint/parser": "^5.42.0",
-    "eslint": "8.22.0",
+    "eslint": "^8.22.0",
     "eslint-config-prettier": "^8.5.0",
     "eslint-plugin-prettier": "^4.2.1",
     "eslint-plugin-react": "^7.31.10",
-    "prettier": "^2.7.1"
+    "husky": "^8.0.1",
```

If you don't use `husky` and `lint-staged`, remove them from dependencies. And also don't forget to remove from `package-lock.json`
```suggestion
    "husky": "^8.0.1",
    "lint-staged": "^13.0.3",
```

---

Done